samples/outlook/90-other-item-apis/get-set-item-subject.yaml

```diff
@@ -0,0 +1,73 @@
+id: outlook-other-item-apis-get-set-item-subject
+name: 'Gets, Sets Item Subject'
+description: 'Gets, sets the subject of a compose item'
+host: OUTLOOK
+api_set: {}
+script:
+    content: |
+        $("#get").click(get);
+        $("#set").click(set);
+
+        function get() {
+            let item = Office.context.mailbox.item as Office.ItemCompose;
+            item.subject.getAsync((result: Office.AsyncResult) => {
+                if (result.status !== Office.AsyncResultStatus.Succeeded) {
+                    console.error(`Action failed with message ${result.error.message}`);
+                    return;
+                }
+                console.log(`Subject is ${result.value}`);
+            })
+        }
+
+        function set() {
+            let item = Office.context.mailbox.item as Office.ItemCompose;
+            let subject = 'Hello World!';
+            item.subject.setAsync(subject, (result: Office.AsyncResult) => {
+                if (result.status !== Office.AsyncResultStatus.Succeeded) {
+                console.error(`Action failed with message ${result.error.message}`);
```

Comment:
nit: formatting - the statements in this if should be indented further

Reply:
fixed locally